promql/engine_test.go

```diff
@@ -545,231 +545,121 @@ load 10s
 func TestMaxQuerySamples(t *testing.T) {
 	test, err := NewTest(t, `
 load 10s
-  metric 1 2
-  bigmetric{a="1"} 1 2
-  bigmetric{a="2"} 1 2
+  metric 1+1x100
```

Comment:
Why did you change this?

Reply:
It was required for testing @ timestamp. Will remove from this PR and add it there.